lib/glueby/wallet.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Glueby
+  class Wallet
+    attr_reader :internal_wallet
+
+    class << self
+      def create
+        new(Glueby::Internal::Wallet.create)
+      end
+
+      def load(wallet_id)
+        new(Glueby::Internal::Wallet.load(wallet_id))
+      end
+
+      def configure(config)
+        case config[:adapter]
+        when 'core'
+          Glueby::Internal::RPC.configure(config)
+          Glueby::Internal::Wallet.wallet_adapter = Glueby::Internal::Wallet::TapyrusCoreWalletAdapter.new
+        else
+          raise 'Not implemented'
+        end
+      end
+    end
+
+    def id
+      @internal_wallet.id
+    end
+
+    # @return [HashMap] hash of balances which key is color_id or empty string, and value is amount
+    def balances
+      utxos = @internal_wallet.list_unspent
+      utxos.inject({}) do |balances, output|
+        script = Tapyrus::Script.parse_from_payload(output[:script_pubkey].htb)
+        key = if script.cp2pkh? || script.cp2sh?
```

Comment:
I think `list_unspent` will return color_id, so we cat use it directly like `output[:color_id]`

Reply:
Add color_id field in response of list_unspent
447b300